src/app/systelab-components/grid/README.md

```diff
@@ -289,6 +289,8 @@ protected getHeaderContextMenuOptions(): Array<GridContextMenuOption<string>> {
 | preferenceName | string | | Preference prefix in order to store the columns size |
 | multipleSelection | boolean | false | Set multiple selection |
 | showChecks | boolean | false | Show a column with a checkbox for each element |
+| noRowsText | string | No Rows To Show | String/Template to display in the grid if the grid hasn't got values (can be html) |
```

Comment:
String/Template to display in the grid is empty (can be html)

Reply:
I have changed the text